cypress/integration/search2.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { visitWithStatusRetries } from "../helpers/visitWithStatusRetries"
+
+describe("Search", () => {
+  it("/search2/results?:query", () => {
+    visitWithStatusRetries("/search2/results?query=andy")
```

Comment:
Yes, we can make the url in the following format: `search2?term=abc`.

Reply:
@dblandin which url format should I do: as Roop (1) suggests or as described in the task in Jira (2)?

Reply:
@dimatretyak Let's go with @anandaroop's suggestion!

Reply:
If we want to support url in `search2?term=banksy` format, then we need to convert `term` query param to `query`, since the `query` key is expected in the `searchState` for [Algolia InstantSearch](https://www.algolia.com/doc/api-reference/widgets/ui-state/react/) (see the attached screenshot)

By this reason, these converters are used:
* converting `term` query param to `query` for Algolia Instantsearch [[link to the source code](https://github.com/artsy/force/blob/02f7229e1a7b523d1cdf6bef8314c4823626769b/src/v2/Apps/Search2/routes/Search2Results/Search2Results.tsx#L58-L59)]
* reverse conversion of query to term when the state of the algolia search changes (for example, during pagination) in order to synchronize the changes with the url [[link to the source code](https://github.com/artsy/force/blob/02f7229e1a7b523d1cdf6bef8314c4823626769b/src/v2/Apps/Search2/routes/Search2Results/Search2Results.tsx#L58-L59)]

@dblandin @anandaroop maybe it's still better to support the url in `search2?query=banksy` format and make redirects for old search links (from external links, bookmarks etc)?

<img width="642" alt="searchState | React InstantSearch widget | Algolia 2022-01-06 19-31-14" src="https://user-images.githubusercontent.com/3513494/148420798-894b976f-8813-458a-9d60-31130ef671ee.png">

Reply:
Thanks for looking into that. I think it's also fine to punt on this for now and leave as-is. We can add a redirect later if it makes sense. (We discussed the possibility of a re-imagined search experience so maybe a simple redirect won't make sense either, but I'll put a ticket on the backlog so we can track it)

Reply:
We discussed it with @anandaroop and decided not to make changes yet and leave `search2/results?query=banksy`